src/lib/utils/SynapseClient.ts

```diff
@@ -1409,6 +1409,49 @@ export const getFileEntityContent = (
   })
 }
 
+/**
+ * Return the FileHandle of the file (latest version) associated to the given FileEntity.
+ * @param sessionToken
+ * @param fileEntityId: ID of the FileEntity
+ * @param endpoint
+ */
+export const getFileEntityFileHandle = (
+  fileEntityId: string,
+  fileEntityVersionNumber?: string,
+  sessionToken?: string,
+): Promise<FileHandle> => {
+  return new Promise((resolve, reject) => {
+    getEntity(sessionToken, fileEntityId, fileEntityVersionNumber).then((entity:Entity) => {
```

Comment:
can add type to getEntity call - `getEntity<FileEntity>` and then remove the cast below